cms/core/migrations/0006_update_user_groups.py

```diff
@@ -0,0 +1,192 @@
+    # Permission = apps.get_model("auth.Permission")
+    # Collection = apps.get_model("wagtailcore.Collection")
+    from django.contrib.auth.models import Group, Permission
+    from wagtail.models import Collection
+
+    group = Group.objects.get(name=settings.PUBLISHING_ADMINS_GROUP_NAME)
+    root_collection = Collection.get_first_root_node()
+
+    for permission in WAGTAIL_PERMISSION_TYPES:
+        GroupCollectionPermission.objects.create(
+            group=group,
+            collection=root_collection,
+            permission=Permission.objects.get(content_type__app_label="wagtailimages", codename=f"{permission}_image"),
+        )
+
+
+def create_snippet_permissions(apps):
+    """Allow Publishing Admins to create, edit and delete snippets."""
+    snippet_classes_dict = {
+        "GlossaryTerm": "core",
+        "ContactDetails": "core",
+        "MainMenu": "navigation",
+        "FooterMenu": "navigation",
+    }
+
+    for model, app in snippet_classes_dict.items():
+        for permission in WAGTAIL_PERMISSION_TYPES:
+            assign_permission_to_group(
+                apps,
+                settings.PUBLISHING_ADMINS_GROUP_NAME,
+                permission_codename=f"{permission}_{model.lower()}",
+                app=app,
+                model=model,
+            )
+        if model in ("MainMenu", "FooterMenu"):
+            # MainMenu and FooterMenu are publishable
+            assign_permission_to_group(
+                apps,
+                settings.PUBLISHING_ADMINS_GROUP_NAME,
+                permission_codename=f"publish_{model.lower()}",
+                app=app,
+                model=model,
+            )
+
+
+def create_bundle_permissions(apps):
+    """Grant all permissions on Bundles to Publishing Admins and Publishing Officers,
+    and view-only access to Viewers.
+    """
+    bundles_app = "bundles"
+    bundle_class = "Bundle"
+
+    for permission in WAGTAIL_PERMISSION_TYPES:
+        for group_name in [settings.PUBLISHING_ADMINS_GROUP_NAME, settings.PUBLISHING_OFFICERS_GROUP_NAME]:
+            assign_permission_to_group(
+                apps,
+                group_name=group_name,
+                permission_codename=f"{permission}_{bundle_class.lower()}",
+                app=bundles_app,
+                model=bundle_class,
+            )
+
+    # View-only access for the Viewers group
+    assign_permission_to_group(apps, settings.VIEWERS_GROUP_NAME, "view_bundle", bundles_app, bundle_class)
+
+
+def create_page_permissions(apps):
+    """Allow Publishing Admins and Publishing Officers all permissions on the HomePage and its children."""
+    # TODO: these shouldn't be imported directly and should use historical models instead
+    from django.contrib.auth.models import Group
+
+    from cms.home.models import HomePage
+
+    # HomePage = apps.get_model("home.HomePage")
+    # Group = apps.get_model("auth.Group")
+
+    home_page = HomePage.objects.first()
```

Comment:
I suggest giving access to the root page `Page.objects.get(depth=1)`. This way they will inherit the same permission for the Welsh copy of the tree